README.md

```diff
@@ -21,16 +21,18 @@ In general, fork the project, create a branch for a specific change and send a
 pull request for that branch. Don't mix unrelated changes. You can use the commit
 message as the description for the pull request.
 
+For more detailed information, see the [contributing page](CONTRIBUTING.md).
 
 ## Running the Unit Tests
 
-Run the unit tests with a local server that supports PHP. No database is required. Pre-configured php local servers are available for Windows and Mac. Here are some options:
-
+Run the unit tests manually with appropriate browsers and any local web server. Here are a selection you could use.
 - Windows: [WAMP download](http://www.wampserver.com/en/)
 - Mac: [MAMP download](http://www.mamp.info/en/index.html)
 - Linux: [Setting up LAMP](https://www.linux.com/learn/tutorials/288158-easy-lamp-server-installation)
 - [Mongoose (most platforms)](http://code.google.com/p/mongoose/)
+- [http-server](https://www.npmjs.com/package/http-server)
 
+You can also run the unit tests inside phantomjs by [installing node](http://nodejs.org/) and the required modules(`npm install`) and then running `grunt test`.
```

Comment:
Let's replace "installing node and the required modules" with "setting up your environment" and linking to CONTRIBUTING.md#environment-minimum-required.